src/controllers/auth_ipa.ts

```diff
@@ -98,14 +102,20 @@ export function SendEmailToRtdHandler(
     if (!isPaFound(errorOrPaInfo)) {
       return ResponseErrorNotFound("Not found", "PA not found in catalogue");
     }
+
     const paInfo = errorOrPaInfo.data;
     const rtdEmail = paInfo.ipa_ou[0].mail_resp;
+    const isSchool = paInfo.ipa_pa[0].tipologia_istat === ISTAT_SCHOOL_TIPOLOGY;
+    // Filter out "da_indicare@x.it"
+    const hasRtd = DUMB_IPA_VALUE_FOR_NULL === rtdEmail;
```

Comment:
are you sure of this check ? (this states that hasRtd is true when the value is "da_indicare@x.it")

Reply:
Thanks @gunzip! Always helpful!